src/src/AdminBundle/Resources/views/Security/login.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends "@AdminBundle/Resources/views/base.html.twig" %}
+
+{% block content %}
+    <div style="margin-top: 100px;" class="col-md-4 offset-md-4">
```

Comment:
inline css sa mi moc nepaci, ale kym nemame rozbehane assety je to asi v pohode, vsak potom to vsetko vytiahneme do suborov

Reply:
To som tam iba zabudol, dal som to tam len preto aby som videl ten login